create_sitemap.py

```diff
@@ -8,6 +8,15 @@
 # We don't want to use settings.LBB_URL to ensure to call the LBB prod server
 LBB_URL = 'https://labonneboite.pole-emploi.fr'
 
+JOBS_SLUG_REGEX = r"^[a-z,-]*$"
```

Comment:
any combination of lower case letters and dashes, correct?

Reply:
Les slugs sont séparés par des virgules donc je teste les slugs (minuscules avec tirets) et il faut que je gère la présence d'une virgule.